opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -949,6 +949,9 @@ class SimulatorFullyImplicitBlackoilEbos
                     if ( active[Water] ) {
                         fluidState.setSaturation(FluidSystem::waterPhaseIdx, saturations[cellIdx*numPhases + pu.phase_pos[Water]]);
                     }
+                    else {
+                        fluidState.setSaturation(FluidSystem::waterPhaseIdx, 0.0);
```

Comment:
Mmh, I do not see any inline here... Maybe this should have its own commit/PR?

Reply:
yeah, that is an unrelated change which I had in my working copy and for which i was too lazy to split up the patch and re-do the jenkins drill once I've noticed that I also commited it. it fixes a "use of uninitialized variable" warning on GCC 7, and I think this may even cause trouble with the satfuncs if the memory for the water saturation happens to be non-0.

Reply:
I hesitate to merge because of this change, while it should only influence oil-gas cases we currently have no such cases in the test suite (I think?). If @totto82 could verify that it works with his gas-oil cases we merge.

Reply:
I repeat my request: does this help or at least not hurt gas-oil @totto82?